### ISSUE TITLE
Changing permissions on a doc causes it to disappear

### ISSUE THREAD

**Issue by bjr70** [MESH-Research/commons#322]
**Describe the bug**
When I edited permissions for this doc to visible for logged in users the doc disappeared from the docs page. It's still in activity, and I can still click on it, but it's not visible from Docs https://hcommons-dev.org/docs/second-document/.

**To Reproduce**
1. Create a doc and then go back to the Docs page.
2. Go back to the doc and click edit
3. Find "Access" and change the Who can read this doc? permissions from "anyone" to "members of Ferdinand's Group"
4. The doc disappears from the folder view and can only be seen in the activity feed. 

This originally did not work and Katie was able to replicate it. Now I'm trying and I can't replicate it. This is very odd. 

**Expected behavior**
The doc remains visible to only members of the group. 

**Context**
Does this bug on production, development, or both?
- [ ] Tested on development instance
- [ ] Occurs on development instance
- [ ] Tested on production
- [ ] Occurs on productions

**Screenshots**

**Additional context**

**Comment by kfitz** [comment]
This didn't happen in my 5.9.2 testing -- I changed permissions on Walter's new doc in Walter's Group to be readable by Walter's Group members only, and it still appears in the group list....

**Comment by bjr70** [comment]
Yeah I can't get it to happen now either. I will try again this afternoon but I'm thinking maybe it was just wonkiness?

**Comment by mikethicke** [comment]
I'm having trouble replicating this, but one thing I did notice was that a doc got kicked into 'awaiting moderation':

![Screen Shot 2022-03-11 at 12.02.29 PM.png](https://images.zenhubusercontent.com/6108aeed64d7503f6ae51eff/d6700980-dd60-429e-93af-f1cdaddf2809)

This causes the doc to be invisible to non-admins, and it can't be fixed from the front end. I don't know how exactly to get a doc into that state. Possibly Akismet is picking up on something? I seem to remember having a similar issue but I can't find searching past issues.

**Comment by mikethicke** [comment]
Ah, this was the issue: https://github.com/MESH-Research/boss-child/issues/113

**Comment by bjr70** [comment]
Oh that's weird. It's not the permissions then, it somehow got knocked into moderation. Thing is I got no notice about moderation or approving it.

**Comment by mikethicke** [comment]
Yeah I think we had similar problems with MESH-Research/boss-child#113 in that we weren't getting notified and things were just disappearing.

I think you probably hit a weird edge case in testing. I haven't been able to replicate. So I'm going to close this and we'll revisit if it comes up for real.
